conf/distro/iot2050-debian.conf

```diff
@@ -35,24 +35,24 @@ PACKAGES_SELECTION ?= "packages-mainline"
 DISTRO_APT_SOURCES_MAINLINE_LIST := "${DISTRO_APT_SOURCES}"
 DISTRO_APT_SOURCES_SNAPSHOT_LIST ?= "conf/distro/debian-snapshot.list"
 
-DISTRO_APT_SOURCES_append_packages-snapshot = " ${DISTRO_APT_SOURCES_SNAPSHOT_LIST}"
-DISTRO_APT_SOURCES_remove_packages-snapshot = "${DISTRO_APT_SOURCES_MAINLINE_LIST}"
+DISTRO_APT_SOURCES:append:packages-snapshot = " ${DISTRO_APT_SOURCES_SNAPSHOT_LIST}"
+DISTRO_APT_SOURCES:remove:packages-snapshot = "${DISTRO_APT_SOURCES_MAINLINE_LIST}"
 
-HOST_DISTRO_APT_SOURCES_append_packages-snapshot = " ${DISTRO_APT_SOURCES_SNAPSHOT_LIST}"
-HOST_DISTRO_APT_SOURCES_remove_packages-snapshot = "${DISTRO_APT_SOURCES_MAINLINE_LIST}"
+HOST_DISTRO_APT_SOURCES:append:packages-snapshot = " ${DISTRO_APT_SOURCES_SNAPSHOT_LIST}"
+HOST_DISTRO_APT_SOURCES:remove:packages-snapshot = "${DISTRO_APT_SOURCES_MAINLINE_LIST}"
 
-# Required as long as openssl is not rebuilt for the host arch
-HOST_DISTRO_APT_PREFERENCES += "conf/distro/preferences.upstream-libssl"
+ISAR_REBUILD_ESSENTIAL_PKGS ?= "openssl"
 
 SDK_INSTALL += "linux-headers-${KERNEL_NAME} mraa"
 SDK_PREINSTALL += "zlib1g-dev:${DISTRO_ARCH} libjson-c-dev:${DISTRO_ARCH}"
 
 SDK_FORMATS = "tar.xz docker-archive"
 
 # used for SWUpdate boot
-IMAGE_CLASSES += "squashfs verity"
 ABROOTFS_IMAGE_RECIPE ?= "iot2050-image-swu-example"
 VERITY_IMAGE_RECIPE ?= "iot2050-image-swu-example"
 INITRAMFS_RECIPE ?= "iot2050-initramfs"
+ABROOTFS_PART_UUID_A ?= "fedcba98-7654-3210-cafe-5e0710000001"
+ABROOTFS_PART_UUID_B ?= "fedcba98-7654-3210-cafe-5e0710000002"
```

Comment:
This should be machine specific, not distro specific.

Reply:
This is not machine specific, it's wic-file specific. So, this does not matter here.